bluepill/src/BPRunner.m

```diff
@@ -19,18 +18,16 @@
 #include <signal.h>
 #include <pwd.h>
 #import <AppKit/AppKit.h>
-#import <BluepillLib/BPSimulator.h>
-#import <BluepillLib/BPCreateSimulatorHandler.h>
-#import <BluepillLib/BPUtils.h>
-#import <BluepillLib/BPWaitTimer.h>
-#import <BluepillLib/SimDeviceSet.h>
-#import <BluepillLib/SimServiceContext.h>
-#import <BluepillLib/SimulatorHelper.h>
-#import <BluepillLib/BPStats.h>
+#import "bp/src/BPSimulator.h"
```

Comment:
what is the binary size & build speed change after this commit ? Using module is supposed to make build faster.

Reply:
Build speed change and binary size changes are negligible. This project is small enough that it's in the noise (testing takes all of our CI time).